crates/y-sweet-core/src/api_types.rs

```diff
@@ -27,13 +22,12 @@ impl Authorization {
 
 #[derive(Deserialize)]
 #[allow(unused)]
+#[serde(default)]
 pub struct AuthDocRequest {
     #[serde(default = "Authorization::full")]
     pub authorization: Authorization,
     #[serde(rename = "userId")]
     pub user_id: Option<String>,
-    #[serde(default)]
-    pub metadata: HashMap<String, Value>,
```

Comment:
We don't use this at present. And if we did, we'd likely need to encode it into the token which we'd like to avoid since its length is unbounded. Let's just drop this field altogether.